src/formats/png/crc.zig

```diff
@@ -0,0 +1,87 @@
+const std = @import("std");
+const io = std.io;
+
+const crc = @This();
+
+var crc_table: ?[256]u32 = null;
+
+fn make_crc_table() void {
+    const magic_n = 0xedb88320;
+
+    crc_table = @as([256]u32, undefined);
+
+    var n: u32 = 0;
+    while (n < 256) : (n += 1) {
+        var c: u32 = n;
```

Comment:
Dunno if `c` and `k` are used in reference CRC32 implementation, but it would be great to have better names for those.